src/main/java/io/streamnative/pulsar/manager/mapper/TopicsStatsMapper.java

```diff
@@ -64,10 +64,10 @@
                 + "sum(msgRateOut) as msgRateOut,"
                 + "sum(msgThroughputOut) as msgThroughputOut,"
                 + "avg(averageMsgSize) as averageMsgSize,"
-                + "sum(storageSize) as storageSize, timestamp FROM topicsStats",
-            "WHERE environment=#{environment} and tenant=#{tenant} and namespace=#{namespace} and timestamp=#{timestamp} and " +
+                + "sum(storageSize) as storageSize, `timestamp` FROM topicsStats",
+            "WHERE environment=#{environment} and tenant=#{tenant} and namespace=#{namespace} and `timestamp`=#{timestamp} and " +
                     "topic IN <foreach collection='topicList' item='topic' open='(' separator=',' close=')'> #{topic} </foreach>" +
-            "GROUP BY cluster, persistent, topic" +
+            "GROUP BY environment, cluster, tenant, namespace, persistent, topic, `timestamp` " +
```

Comment:
this "group by" clause was incomplete and Apache Calcite wasn't happy

Reply:
I fixed it here https://github.com/streamnative/pulsar-manager/pulls